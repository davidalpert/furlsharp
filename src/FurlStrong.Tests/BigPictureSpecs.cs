using ApprovalTests;
using FurlStrong.Generation;
using FurlStrong.Internal;
using NUnit.Framework;

namespace FurlStrong.Tests
{
    [TestFixture]
    public class BigPictureSpecs
    {
        [Test]
        public void When_generating_it_should_look_right()
        {
            var urlmap = ManifestResourceHelper.ExtractResourceToString("FurlStrong.Tests.sample.urls");

            var generator = new Generator();

            var result = generator.GenerateStrongUrls(urlmap);

            Approvals.Verify(result);
        }
    }
}
