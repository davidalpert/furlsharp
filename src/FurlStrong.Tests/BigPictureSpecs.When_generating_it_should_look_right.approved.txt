
namespace FurlStrong.Sample {

<< Generated Code Goes Here >>

}
